### PR TITLE
Lots of improvement on testing feedback suggestions

### PR DESCRIPTION
**Pull request by pal03377**
Documentation can now be found in `inject-testing-data/README.md`. Also, there is a new format for the test submissions that makes it much easier to try things out. 

Thanks @botrufus for helpful feedback about how participation IDs should be handled! It's much cleaner now.

### CODE REVIEW

**Comment by pal03377** [comment]
Apparently, I did something wrong about branching (branched off from old version?), but I resolved the conflicts now

**Review by crustypluto19 — APPROVED** [review]
Great work! The readME file looks real professional already 😄

File: feedback-suggestion/src/app/endpoints/feedback_suggestions.py [hunk 1]
```diff
@@ -46,13 +46,25 @@ def get_feedback_suggestions(request: FeedbackSuggestionsRequest):
...
+                        original_code = feedback.code
+                        feedback_to_give = dict(feedback)
+                        feedback_to_give["code"] = method.get_source_code()
+                        feedback_to_give["from_line"] = method.get_start_line()
+                        feedback_to_give["to_line"] = method.get_stop_line()
+                        suggested_feedbacks.append({
+                            "feedback": feedback_to_give,
+                            "originally_on_code": original_code,
+                            "similarity_score": similarity_score
+                        })
```
Comment (botrufus) [inline feedback-suggestion/src/app/endpoints/feedback_suggestions.py]: Why send the code twice? Technically we don't need the code at all in Themis. The same goes for the similarity score. I would just send what really is needed, and that is the feedback.

Comment (pal03377) [reply feedback-suggestion/src/app/endpoints/feedback_suggestions.py]: One is the code from the submission in question, the other is the code from the feedback in the DB. They are the two different pieces of code compared by codeBERT. This is useful for debugging, even though our app does not need it. I will add a boolean parameter which decides whether to include the code and set it to `False` by default.

**Review by botrufus — APPROVED** [review]

**Review by huber-florian — APPROVED** [review]

**Pull request merged**